frontend/packages/console-shared/src/utils/order-extensions.ts

```diff
@@ -0,0 +1,84 @@
+interface ItemsToSort {
```

Comment:
These utilities are copied from add-page-utils

[TODO]: refactor utils to create common utils which can be used to order the extensions